scm/src/scm_input.F90

```diff
@@ -770,6 +770,16 @@ subroutine get_case_init(scm_state, scm_input)
   scm_input%input_vegsrc   = input_vegsrc
   
   scm_input%input_slmsk    = input_slmsk
+  scm_input%input_canopy   = input_canopy
```

Comment:
These changes came from #364, but are not needed for this PR, right?

Reply:
Yes. I cherry picked a few files from #364 to create this PR, which was a bit too much for one PR IMO. I didn't want to remove the UFS-replay changes manually, so it just left this stuff in there. A new PR for UFS-replay is imminent anyways.